src/lbaf/Model/lbsRank.py

```diff
@@ -59,6 +59,9 @@ def copy(self, rank):
         self.__sentinel_objects = copy.copy(rank.__sentinel_objects)
         self.__migratable_objects = copy.copy(rank.__migratable_objects)
 
+    def __lt__(self, other):
```

Comment:
I am not sure that we want this to be always the comparison operator between ranks, as they can be sorted by other QoIs (like index, work, etc.)

Reply:
Leaving this as default comparison but creating follow-on issue for later discussion of how we want to handle this in general (and be consistent with vt in this regard)